### PR TITLE
Random phase

### PR DESCRIPTION
**Pull request by mbillingr**
The doctest example of `randomize_phase` performed randomizaton along the wrong axis. (Did this happen when changing data orientation?)

### CODE REVIEW

File: scot/datatools.py [hunk 1]
```diff
@@ -144,14 +144,14 @@ def randomize_phase(data):
...
+        s = np.sin(np.linspace(0,10*np.pi,1000))
+        x = np.vstack([s, np.sign(s)])
         y = randomize_phase(x)
         subplot(2,1,1)
         title('Phase randomization of sine wave and rectangular function')
-        plot(x), axis([0,1000,-3,3])
+        plot(x.T + [1.5, -1.5]), axis([0,1000,-3,3])
         subplot(2,1,2)
-        plot(y), axis([0,1000,-3,3])
+        plot(y.T + [1.5, -1.5]), axis([0,1000,-3,3])
```
Comment (cle1109) [inline scot/datatools.py]: Why did you introduce `[1.5, -1,5]` here?

**Comment by cle1109** [comment]
Can you rebase to master?

File: scot/datatools.py (hunk 1 above)
Comment (mbillingr) [inline scot/datatools.py]: To avoid overlap of the functions.

**Comment by cle1109** [comment]
LGTM - +1 for merge

**Comment by mbillingr** [comment]
I wonder why coverage decreases in this PR. According to coveralls, we miss 13 additinoal lines in `_infomax.py`, although we didn't touch this file or related tests.

**Comment by cle1109** [comment]
RIght, no idea either. However, we could test extended Infomax, which we currently miss out on.

**Pull request merged**